tests/e2e/specs/modules/analytics/setup-proxy-no-account-no-tag.test.js

```diff
@@ -75,7 +75,7 @@ describe( 'setting up the Analytics module with no existing account and no exist
 		await expect( page ).toClick( '.googlesitekit-analytics__select-country' );
 		await expect( page ).toClick( '.mdc-menu-surface--open li', { text: /united kingdom/i } );
 
-		await expect( page ).toMatchElement( 'p', { text: /need to give Site Kit permission to create an Analytics account/i } );
+		await expect( page ).toMatchElement( 'p', { text: /need to give Site Kit permission to create an Analytics account/i, timeout: 750 } );
```

Comment:
That seems unrelated - did that fix a random test failure you noticed while working on this issue?

Reply:
Yeah, that helped with a weird issue, but now it seems to be not needed anymore. Reverted.